projects/js-packages/react-data-sync-client/README.md

```diff
@@ -1,8 +1,18 @@
-# react-data-sync-client
+# React DataSync Client
 
-DataSync client for React
+## Debug
 
-## How to install react-data-sync-client
+WP JS DataSync has a debug mode that can be used to disable certain DataSync values. Read more about it [here](../../packages/wp-js-data-sync/README.md#debug).
+
+Disabling DataSync values via the WP JS DataSync package will disable them on the first page load, but not on the endpoints. You can ask React DataSync Client to disable certain endpoints by adding a `#ds-debug-disable` hash to the URL.
+
+To disable a DataSync endpoint request, add `#ds-debug-disable=<endpoint-key>` to the URL.
+To disable all DataSync endpoint requests, add `#ds-debug-disable=all` to the URL.
+
+#### Debug Example
+If your dashboard URL is `https://example.com/wp-admin/admin.php?page=example`, and you want to disable the `widget_status` endpoint, you would navigate to `https://example.com/wp-admin/admin.php?page=example#ds-debug-disable=widget_status`.
+
+You can also disable all endpoints by adding `#ds-debug-disable=all` to the URL.
```

Comment:
`ds-debug-disable` isn't going to prevent making requests - it's more meant as an useful tool to quickly mocking an error response, allowing us to test that missing a value doesn't kill the dashboard.

Reply:
Oh sure. It just says that it disables endpoint requests 😅 

> To disable a DataSync endpoint request, add `#ds-debug-disable=<endpoint-key>` to the URL.
> To disable all DataSync endpoint requests, add `#ds-debug-disable=all` to the URL.